zipkin-collector/scribe/src/main/java/zipkin2/collector/scribe/ScribeInboundHandler.java

```diff
@@ -162,7 +162,7 @@ void maybeReadPayload(ChannelHandlerContext ctx) {
           buf.release();
         }
       } else {
-        returned.writeBytes(content.array(), content.offset(), content.length());
+        returned.writeBytes(content.array(), 0, content.length());
```

Comment:
Just `writeBytes(content.array())`

Back to my original, somewhat broken version ;)

Reply:
:)